addons/web/static/src/views/fields/properties/properties_field.scss

```diff
@@ -115,6 +104,15 @@
     .oi-draggable {
         cursor: url(/web/static/img/openhand.cur), grab;
     }
+    // mobile view
```

Comment:
Not necessary and a bit biased, as it's meant to be responsive and not specifically for mobile, would remove ^^